.github/workflows/tests.yml

```diff
@@ -73,11 +74,33 @@ jobs:
           # a warning on Python 3.10
           # Ignore DeprecationWarnings raised by cvxpy importing scipy.sparse.X
           # under SciPy 1.8.0+.
+          # Ignore DeprecationWarnings raised by versions of
+          # setuptools >= 65.0.0 during pyximport imports This can be removed
+          # once https://github.com/cython/cython/issues/4985
+          # is fixed and released.
           - case-name: Python 3.10
             os: ubuntu-latest
             python-version: "3.10"
             condaforge: 1
-            pytest-extra-options: "-W ignore::ImportWarning -W ignore::DeprecationWarning:cvxpy.interface.scipy_wrapper"
+            pytest-extra-options: "-W ignore::ImportWarning -W ignore::DeprecationWarning:cvxpy.interface.scipy_wrapper -W ignore:Absolute:DeprecationWarning"
```

Comment:
We should see what warning ignores we can drop here, but definitely for later PRs.